src/wp-login.php

```diff
@@ -619,7 +619,7 @@ function wp_login_viewport_meta() {
 			}
 
 			if ( $remind_interval > 0 ) {
-				update_option( 'admin_email_lifespan', time() + $remind_interval );
+				update_option( 'admin_email_lifespan', time() + $remind_interval, false );
```

Comment:
This still needs to be reverted per https://github.com/WordPress/wordpress-develop/pull/6465#discussion_r1731915673.
```suggestion
				update_option( 'admin_email_lifespan', time() + $remind_interval );
```